sdk/include/reactos/shellutils.h

```diff
@@ -609,34 +609,39 @@ struct CDataObjectHIDA
     explicit CDataObjectHIDA(IDataObject* pDataObject)
         : m_cida(nullptr)
     {
-        m_medium.tymed = TYMED_NULL;
+        m_hr = CreateCIDA(pDataObject, &m_cida, m_medium);
+    }
+
+    ~CDataObjectHIDA()
+    {
+        DestroyCIDA(m_cida, m_medium);
+    }
 
+    static void DestroyCIDA(CIDA *pcida, STGMEDIUM &medium)
+    {
+        if (pcida)
+            ::GlobalUnlock(pcida);
```

Comment:
Is `pcida` a global handle?

Reply:
No, of course not.

Reply:
Why use `GlobalUnlock` for non-global handle?

Reply:
> Why use `GlobalUnlock` for non-global handle?

[TYMED_HGLOBAL is GMEM_MOVEABLE](https://learn.microsoft.com/en-us/windows/win32/api/objidl/ne-objidl-tymed)

Reply:
You can use `GlobalHandle` function.
https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-globalhandle

Reply:
1. Why are you looking for problems where there are none?
2. I did not write this, the code already did this successfully for how many years?

Reply:
Ok, let's dig a hole and bury it in there.
This is not a problem.

Reply:
I tried to not change the original code but I can make it unlock on the HGLOBAL if that makes you feel better.

Reply:
pls bury it in the ground. This is a million year secret.

Reply:
`GlobalUnlock` has return value of `BOOL`.


(You don't have to fix it. I'm just saying)

Reply:
`ReleaseStgMedium` will call `GlobalFree` and `GlobalFree` will free the memory even if it is locked but like I said, I will change this to follow the documentation exactly.